lib/routes.php

```diff
@@ -88,6 +88,7 @@
  *  description="Schedule"
  * )
  */
+$app->get('/schedule/grid/data', ['\Xibo\Controller\Schedule','grid'])->setName('schedule.grid.data');
```

Comment:
Is there scope for this to be just `/schedule` and be called `schedule.search` so that it is the same as the other GET's?